src/config.rs

```diff
@@ -665,6 +666,12 @@ fn validate_config(cfg: &ConfigItems) -> Result<(), Error> {
         err!(format!("`DATABASE_MAX_CONNS` contains an invalid value. Ensure it is between 1 and {}.", limit,));
     }
 
+    if let Some(log_file) = &cfg.log_file {
+        if OpenOptions::new().append(true).create(true).open(log_file).is_err() {
```

Comment:
I suggest to use `std::fs::OpenOptions` here instead of making it global for only this function.

```suggestion
if std::fs::OpenOptions::new().append(true).create(true).open(log_file).is_err() {
```